test/run

```diff
@@ -378,25 +378,15 @@ else
 fi
 
 if $HOST_OS_APPLE; then
-    # Grab the developer directory from the environment or try xcode-select
-    if [ "$XCODE_DEVELOPER_DIR" = "" ]; then
-      XCODE_DEVELOPER_DIR=`xcode-select --print-path`
-      if [ "$XCODE_DEVELOPER_DIR" = "" ]; then
-        echo "Error: XCODE_DEVELOPER_DIR environment variable not set and xcode-select path not set"
+#   if somebody has the skill to deal with miltiple xcode versions/releases
```

Comment:
I don't quite understand this comment. Do you mean there's a risk that `$SDKROOT` isn't set to an absolute path?